TPP.Core/Commands/Definitions/PollCommands.cs

```diff
@@ -0,0 +1,100 @@
+            //Options can be in ID or String form but must all match
+            //Multi-answer polls use spacing between options
+            //!vote IDENTIFIER Option2 Option4
+            //!vote IDENTIFIER 2 4
+
+            (string pollCode, ManyOf<string> voteStrs) = await context.ParseArgs<string, ManyOf<string>>();
+            ImmutableList<string> votes = voteStrs.Values;
+
+            Poll? poll = await _pollRepo.FindPoll(pollCode);
+            if (poll == null)
+                return new CommandResult { Response = $"No poll with the code '{pollCode}' was found." };
+
+            List<int> selectedOptions = new();
+            foreach (string voteStr in votes)
+            {
+                PollOption? option = null;
+                if (int.TryParse(voteStr.TrimStart('#'), out int voteInt))
+                    option = poll.PollOptions.FirstOrDefault(o => o.Id == voteInt);
+                option ??= poll.PollOptions.FirstOrDefault(o => o.Option == voteStr);
+                if (option == null)
+                    return new CommandResult { Response = $"Invalid option '{voteStr}' included for poll '{pollCode}'." };
+                selectedOptions.Add(option.Id);
+            }
+
+            VoteFailure? failure = await _pollRepo.Vote(
+                pollCode, context.Message.User.Id, selectedOptions.ToImmutableList());
+            return new CommandResult
+            {
+                Response = failure switch
+                {
+                    null => "Successfully voted.",
+                    VoteFailure.PollNotFound => $"No poll with the code '{pollCode}' was found.",
+                    VoteFailure.PollNotAlive => "The poll has already ended.",
+                    VoteFailure.AlreadyVoted => "You already voted in that poll.",
+                    VoteFailure.CannotVoteForNone => poll.MultiChoice
+                        ? "Must vote for at least one option."
+                        : "Must vote for an option.",
+                    VoteFailure.NotMultipleChoice => "Cannot select multiple options in non-multi-choice polls.",
+                    VoteFailure.InvalidOptions { Options: var options } =>
+                        $"Invalid poll options: {string.Join(", ", options)}.",
+                    _ => throw new ArgumentOutOfRangeException(nameof(failure), "Unhandled poll voting result")
+                }
+            };
+        }
+
+        public async Task<CommandResult> Poll(CommandContext context)
+        {
+            string pollCode = await context.ParseArgs<string>();
+            Poll? poll = await _pollRepo.FindPoll(pollCode);
+            if (poll == null)
+                return new CommandResult { Response = $"No poll with the code '{pollCode}' was found." };
+
+            string Percentage(PollOption option) => poll.Voters.Count == 0
+                ? "0" // avoid division by zero
+                :$"{100 * (option.VoterIds.Count / (double) poll.Voters.Count):0.#}";
```

Comment:
two errors on this line, please fix

Reply:
I don't see errors, but I suppose you mean non-conforming formatting

Reply:
both errors are visible here
https://github.com/TwitchPlaysPokemon/tpp-core/pull/217/files#diff-ad784caac43714628eca51b1aaf20f75f3a6242c547fafd2429cd3867a694906R91